app/models/account.rb

```diff
@@ -63,6 +63,14 @@ def self.single_tenant_default
     end
   end
 
+  # @return [Boolean] whether this Account is the global tenant in a multitenant environment
+  def self.global_tenant?
+    # Global tenant only exists when multitenancy is enabled and NOT in test environment
+    # (Test environment always acts like a single tenant, i.e. tenant switching not possible)
```

Comment:
This is a partially misleading statement about test behavior.  See [multitenancy_metadata.rb#L12-L22](https://github.com/samvera-labs/hyku/blob/master/spec/support/multitenancy_metadata.rb#L12-L22) for more detail.  In short, there are many ways tests do **not** behave like single tenant: the most common default is `:multitenant`, and `:singletenant` is the exception!